kernels/volk/volk_16i_32fc_dot_prod_32fc.h

```diff
@@ -147,8 +147,7 @@ static inline void volk_16i_32fc_dot_prod_32fc_u_sse(lv_32fc_t* result,
     unsigned int number = 0;
     const unsigned int sixteenthPoints = num_points / 8;
 
-    float res[2];
-    float *realpt = &res[0], *imagpt = &res[1];
+    lv_32fc_t returnValue;
```

Comment:
Compilers would potentially complain about uninitialized variables here.
I suggest: `lv_32fc_t returnValue = lv_cmake(0.0f, 0.0f);`
I'd update the other occurrences as well.

Reply:
I'm not sure compilers would be confused by this particular case, but it would make the code easier to understand for humans. I'll add initializations and change all the updates to `+=`.